server/dive_utils/constants.py

```diff
@@ -8,7 +8,7 @@
 
 webValidImageFormats = {"png", "jpg", "jpeg"}
 validImageFormats = {*webValidImageFormats, "tif", "tiff", "sgi", "bmp", "pgm"}
-validVideoFormats = {"mp4", "avi", "mov", "mpg", "mpeg", "wmv", "flv"}
+validVideoFormats = {"mp4", "avi", "mov", "mpg", "mpeg", "wmv", "flv", "webm"}
```

Comment:
I notice there are some differences with:
https://github.com/Kitware/dive/blob/main/client/dive-common/constants.ts
(probably my fault)
Should we see this as an opportunity to sync them up so we have the same mime and file extension support?

Reply:
Sure, I'll do that.